### PR TITLE
added testcase for fields command

### PR DESCRIPTION
**Pull request by pkess**
Here's my second test. I want to check the command "fields" of beets. I think the coverage of this would be ok now, but at the moment i don't check the output of the program. So i don't really want you to merge this branch at the moment.

Do you have any suggestions, how we should test it? I think it would not make sense to test it against a fixed list of text.

### CODE REVIEW

File: test/test_ui_commands.py [hunk 1]
```diff
@@ -84,6 +84,25 @@ def test_query_album(self):
...
+    def setUp(self):
+        super(FieldsTest, self).setUp()
+
+        self.io.install()
+
+        self.libdir = os.path.join(self.temp_dir, 'testlibdir')
+        os.mkdir(self.libdir)
+
+        # Add a file to the library but don't copy it in yet.
+        self.lib = library.Library(':memory:', self.libdir)
```
Comment (sampsyo) [inline test/test_ui_commands.py]: If you inherit from `_common.LibTestCase` instead of plain `_common.TestCase`, you'll get `self.lib` for free.

**Comment by sampsyo** [comment]
Looking good! Yes, checking against a fixed list would work. To make the test slightly more robust to changes, though, it may be a good idea to read out `Item._fields` for the ground truth on fixed attributes (so things don't break every time we add a new one).

It might also be nice to check the display of plugin-provided fields, which would involve cooking up a dummy plugin for the test.

**Comment by pkess** [comment]
ok, i just added a method to test the output (not very strong, but robust against changes in output)

**Comment by sampsyo** [comment]
Looks ideal! Thank you again! :sparkles:

**Pull request merged**